content/_releases/22/28.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Product Update - v22.28
+description: Details of the product update v22.28 from 14th of July 2022.
+layout: rel
+version: "22.28"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-07-14
+redirect_from:
+  - /releases/22.28.html
+---
+
+## Improvements and Updates
+
+### Per-tenant frontend styles feature removed
+
+We've removed feature that builds per-tenant styles and applied them based on `tenant.cssEnabled` and `tenant.code property`. We also removed all tenant styles from our codebase.
```

Comment:
removed **the** feature.

In general we should explain the impact for Users / Tenant Owners or Admins in a third sentence - i.e. how does this help them in applying or regulating the tenant UI?